lib/Conversation/index.js

```diff
@@ -58,7 +60,8 @@ class Conversation extends Component {
     resolved: false,
     handleUndoResolve: null,
     onSubscribeChange: null,
-    isSubscribed: false
+    isSubscribed: false,
+    cornerIcon: ''
```

Comment:
I would maybe default this to `null` rather than an empty string, to rule out any confusion.

Reply:
Agree, and done